xcube_cci/chunkstore.py

```diff
@@ -517,6 +530,9 @@ def get_time_ranges(self, dataset_id: str, cube_params: Mapping[str, Any]) -> Li
             this = next
         return request_time_ranges
 
+    def get_default_time_range(self):
+        return (self._metadata['temporal_coverage_start'], self._metadata['temporal_coverage_end'])
```

Comment:
Are we absolutely sure, that keys  `temporal_coverage_start` and `temporal_coverage_end` are always present? Otherwise we risk a `KeyError` here!